src/tpm2_linux.c

```diff
@@ -173,10 +173,8 @@ int TPM2_LINUX_SendCommand(TPM2_CTX* ctx, TPM2_Packet* packet)
         #endif
             rc = TPM_RC_FAILURE;
         }
-
-        close(fd);
```

Comment:
The device does need "closed" at some point. Please make sure the wolfTPM cleanup closes it.